Makefile

```
@@ -143,6 +143,12 @@ fmt:
 vet:
 	$(GO_CMD)  vet ./...
 
+TEST ?= ./...
```

Just a thought/suggestion, not a blocker. I think a very common pattern is to call this kind of variable `WHAT`. Also, then it could be used in some other targets like `vet:` above.

```suggestion
WHAT ?= ./...
```

WDYT? Or alternatively suggestion to change the name slightly:

```suggestion
TESTS ?= ./...
```

---

I would change it to TESTS. So far from my experience (not that extensive :), i have seen the TEST/TESTS being used , but have yet to see WHAT

---

fixed